src/test/java/POM/pages/AlbumsPage.java

```diff
@@ -10,8 +12,10 @@ public class AlbumsPage extends BasePage{
 
     @FindBy(xpath = "//section[@id='albumsWrapper']//a[contains(text(),'Makaih')]")
     WebElement artistMakaih;
-    @FindBy(xpath = "//button[contains(@title,'Like Pluto')]")
+    @FindBy(xpath = "//div[@class='song-list-wrap main-scroll-wrap album']//button[contains(@title,'Like Pluto')]")
     WebElement heartBtnPluto;
+    @FindBy(xpath = "//button[@title='Like BornKing by Makaih Beats']")
+    WebElement heartBtnBornKing;
 
     String urlAlbumsPage = "https://bbb.testpro.io/#!/albums";
```

Comment:
**The private access modifier is important because it allows encapsulation and information hiding, which are core principles of object-oriented programming.** 

```
    @FindBy(xpath = "//section[@id='albumsWrapper']//a[contains(text(),'Makaih')]")
    private WebElement artistMakaih;
    @FindBy(xpath = "//div[@class='song-list-wrap main-scroll-wrap album']//button[contains(@title,'Like Pluto')]")
    private WebElement heartBtnPluto;
    @FindBy(xpath = "//button[@title='Like BornKing by Makaih Beats']")
    private WebElement heartBtnBornKing;
    private String urlAlbumsPage = "https://bbb.testpro.io/#!/albums";
```